common/filters/args.go

```diff
@@ -0,0 +1,68 @@
+package filters
+
+import (
+	"encoding/json"
+)
+
+type Args struct {
```

Comment:
I'm not sure about this abstraction for a high level interface.
I know it's just copied from the docker code base.

Also, let's not add a `common` package since it's not conveying any real information.

Reply:
this filter is not only used in `ContainerList`, but also in `ImageList`, where should I place such types? duplicating these types doesn't sound to be a good idea.

and in moby/moby, these types are located at https://github.com/moby/moby/blob/master/api/types/, not sure whether it is a better choice.